tests/Unit/Entity/EditableOrderTraitTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Tests\Unit\Entity;
+
+use PHPUnit\Framework\TestCase;
+use Setono\SyliusOrderEditPlugin\Tests\Application\Entity\Order;
+
+final class EditableOrderTraitTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function it_sets_initial_total(): void
```

Comment:
Also, we should probably talk about the convention, as in other tests I use a standard `public function testIt***` method names, instead of spec-like `function it_***` with a `@test` annotation. Which one you prefer?